.github/CHANGELOG.md

```diff
@@ -50,6 +51,7 @@ See full commit details ...
 * Pandas is removed from the package requirements. [#63](https://github.com/XanaduAI/flamingpy/pull/63)
 * `mpi4py` is **not** a development requirement for Linux users. [#64](https://github.com/XanaduAI/flamingpy/pull/64)
 * CI test check that code executes properly with and without MPI. [#64](https://github.com/XanaduAI/flamingpy/pull/64)
+* Added tests for `EGraph` plots. [#60](https://github.com/XanaduAI/flamingpy/pull/60)
```

Comment:
This should be added to the section **Release 0.8.2a5 (development release)** above.